components/locale/Switcher.vue

```diff
@@ -0,0 +1,41 @@
+<script setup>
+  const { locale, locales } = useI18n()
+  // generates URLs to switch between locales
+  const switchLocalePath = useSwitchLocalePath()
+  const availableLocales = computed(() => {
+    // returns a list of locales, exluding current locale
+    return locales.value.filter((i) => i.code !== locale.value)
+  })
+  const currentLocale = computed(() => {
+    return locales.value.filter((i) => i.code === locale.value)
+  })
+</script>
+
+<template>
+  <VMenu transition="scale-transition">
+    <!-- activator btn -->
+    <template #activator="{ props }">
+      <VBtn
+        v-bind="props"
+        rounded="lg"
+        prepend-icon="mdi-translate"
+        append-icon="mdi-chevron-down"
```

Comment:
Think we can remove the arrow here.

Reply:
I think I would prefer to either have the full language name written out (`language.name`) and/or have a (flat) flag icon for the corresponding language instead of the symbol-icon which is there now. I feel like that it is intuitive enough with the flag to understand it is a locale switcher, but with the symbols, we might still need the chevron (or at least have a clear box shadow in the VMenu).

Something like this could be nice...
![image](https://github.com/et-dagen/et-dagen/assets/26115581/03370fab-e285-415c-a5f0-110e96cdb768)

Reply:
have changed it a bit now, plz check if ok c: